client/components/App.jsx

```diff
@@ -1,29 +1,23 @@
 import React from 'react'
+import { connect } from 'react-redux'
 
-import { getFruits } from '../apiClient'
+import { fetchFruits } from '../actions'
 
-class App extends React.Component {
+export class App extends React.Component {
   state = {
     fruits: []
   }
 
   componentDidMount () {
-    getFruits()
-      .then(fruits => {
-        this.setState({ fruits })
-        return null
-      })
-      .catch(() => {
-        console.log('Error getting fruit.')
-      })
+    this.props.dispatch(fetchFruits())
   }
 
   render () {
     return (
       <div className='app'>
         <h1>Fullstack Boilerplate - with Fruits!</h1>
         <ul>
-          {this.state.fruits.map(fruit => (
+          {(this.props.fruits).map(fruit => (
```

Comment:
@rossjourdain not sure if you're still editing this repo, but if you are, we don't need the parenthesis around `this.props.fruits`.

Reply:
Good call, yeah that's carried forward from the original.  I'll fix it.